.circleci/config/jobs/algolia_index.yml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - run:
+      name: Push content to Algolia Index
+      command: |
+        cd website/
+        npm i
+        node scripts/index_search_content.js
```

Comment:
I'm sure there's a cleaner way to set this sort of thing up. Happy to clean it up. Also, do we have access to Node.js here or is there a prerequisite step where I'd have to set up Node.js first?